src/stylesheets/common/_helpers.scss

```diff
@@ -172,14 +172,13 @@
   color: #333333;
 }
 
-.text-16 {
-  // font-weight: normal;
-  // font-size: 16px;
-  // line-height: 1.6;
+.deemphasize-text {
```

Comment:
I've seen this stock Bootstrap class used, that sounds like it's used in a very similar way: https://github.com/mapzen/styleguide/blob/77b0d5cb85ad8f08d56ff1a840182a415902d7eb/src/stylesheets/bootstrap/_type.scss#L101-L104

There is an example of `.text-muted` in use on the Settings page, under the Payment method heading where it says "(We won't charge you without your permission)". The color is #777777, which is the Bootstrap default for this class.

Should we rely on `.text-muted` instead (and adjust the text color to match our desired de-emphasized color)? Or do we have two levels of de-emphasized text (in which case we might want to use the same naming conventions, and document the difference between the two)?